public/javascript/bot.js

```diff
@@ -53,12 +54,19 @@ function listenerCallback(err, event) {
 // Bot handlers
 function messageHandler(event) {
 	var message = event.body;
-	if (message.includes("@meme")) {
-		rickroll(userAPI, event.threadID);
-	} else if (message.match(/^@color \#[0-9A-Fa-f]{6}$/)) {
-		setChatColor(userAPI, event.threadID, message)
+	if (message != null) {
+		// rickroll
+		if (message.includes("@meme")) {
+			rickroll(userAPI, event.threadID);
+		// setChatColor
+		} else if (message.match(/^@color \#[0-9A-Fa-f]{6}$/)) {
+			setChatColor(userAPI, event.threadID, message);
+		// getWeather
+		} else if ((/^@weather ([0-9]{5}|[a-zA-z ]+)$/).test(message)) {
```

Comment:
You should include `,` in the city name regex as well. (e.g.: San Francisco, CA)